frontend/src/components/pages/overview/Overview.tsx

```diff
@@ -321,7 +321,9 @@ function ClusterDetails() {
 
   const serviceAccounts = overview.redpanda?.userCount ?? 'Admin API not configured';
 
-  const aclCount = overview.kafkaAuthorizerInfo?.aclCount ?? 'Authorizer not configured';
+  const aclCount = overview.kafkaAuthorizerInfo
+    ? overview.kafkaAuthorizerInfo.aclCount
+    : (overview.kafkaAuthorizerError?.message ?? 'Authorizer not configured');
```

Comment:
We should work with the error codes we return in connectrpc responses. Possibly we should tackle this as an initiative but these error codes have been introduced so that we have more control over how we can present errors in the UI.

Reply:
I'm working on a new way of data fetching right now, so I'll take this into consideration.

Reply:
@jvorcak you can check how we render toasts and how queries/mutations in custom hooks handle errors